### ISSUE TITLE
Update Angular 11 : impossible to build cause of "no such file" icons.css

### ISSUE THREAD

**Issue by sabrinaToulouse** [swimlane/ngx-datatable#1999]
**I'm submitting a ...** (check one with "x")

```
[ x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

I just updated to angular 11.2.14 (and ngx-datatable ~19.0.0), and when I try to build my app I now get the error : 

ERROR: ENOENT: no such file or directory, open 'C:\Users\...\src\tables\component\table-list\~@swimlane\ngx-datatable\assets\icons.css'
I've imported the css like : 
```
@import "~@swimlane/ngx-datatable/index.scss";
@import "~@swimlane/ngx-datatable/themes/material.scss";
@import '~@swimlane/ngx-datatable/assets/icons.css';
```

I had same issue with the others, but then I used .scss files instead of .css and it solved the issue, but there's no .scss for the icons file.

Be able to build the project

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

- **Table version:**

- **Angular version:**

- **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

- **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by LucasDshg** [comment]
Hi. I think your import is incorrect because the path error is no get in the node module folder.

I have it install em Angular 11 and 12 end working fine.

**Comment by ScottSpittle** [comment]
> Hi. I think your import is incorrect because the path error is no get in the node module folder.
> 
> I have installed it in Angular 11 and 12 end working fine.
> 
> My imports.
> 
> ```css-scss
> @import "~@swimlane/ngx-datatable/index.css";
> @import "~@swimlane/ngx-datatable/themes/material.scss";
> @import "~@swimlane/ngx-datatable/assets/icons.css";
> ```

You should be using the style import in angular.json https://angular.io/guide/workspace-config#styles-and-scripts-configuration

**Comment by LucasDshg** [comment]
Yes! I forgot to say this because i impoterd i my ower lib using @import and in my apps just rum npm install 😁.
In this case i dont need to import in angular.json.

If want to change variavel on the style u can import in angular.json and add @import after your changes. This work with bootstrap. Make sure this lib accept it.
